OneCommunityGlobal/HighestGoodNetworkApp
# Shefali_Unit_Test_Correct_Test_Folder

# Description
Unit test code refactored( added test file in corresponding __tests__ folder)

## How to test:
1. check into current branch
2. npm install

Then run all below tests(16 tests):
npm test Checkbox.test.js
![Checkbox test file](https://github.com/user-attachments/assets/663cc44d-f568-4dc7-b3ee-67b45db59a5a)

npm test CheckboxCollection.test.js
![CheckboxCollection test file](https://github.com/user-attachments/assets/0f6c3aef-d548-46f0-827e-ad4cf3959061)


npm test FileUpload.test.jsx
![FileUpload test file](https://github.com/user-attachments/assets/2a209a98-1d95-494a-987d-e244d5a93545)

npm test Input.test.jsx
![input test file](https://github.com/user-attachments/assets/0f4df540-eecc-40d8-b65e-010b850ee1ab)

npm test SkeletonLoading.test.jsx
![SkeletonLoading test file](https://github.com/user-attachments/assets/41ed4073-9aa8-4cac-b848-76a77ceb7ffe)


npm test Dropdown.test.jsx
![Dropbox test file](https://github.com/user-attachments/assets/21e8933c-81c3-49ad-ad9e-6bb79e03e3be)


npm test Modal.test.jsx
![Modal test file](https://github.com/user-attachments/assets/eca7366d-56e3-47b9-89f4-2a98068c2ffd)


npm test Paging.test.jsx
![Paging test file](https://github.com/user-attachments/assets/a08e2f81-fdc8-4453-bc74-a757c327c510)


npm test Member.test.jsx
Note : Even if you see console error here in this test, it is not related to this PR, code is just refactored here so functionality remains same.So it is okay to get this error and raise a separate PR and work on this if anything needs to be done here.

![Member test file](https://github.com/user-attachments/assets/337e6c02-5610-4c20-ab93-4274129bb0f8)

npm test TinymceEditor.test.jsx
![TinymceEditor test file](https://github.com/user-attachments/assets/0bfeaa9f-e853-4efe-ad58-5d734c6e4489)


npm test AssignTableRow.test.js 
![AssignTablerow test file](https://github.com/user-attachments/assets/10e1ce4f-5736-47ba-8013-2fb037c41c05)


npm test FoundUser.test.jsx

![FoundUser test file](https://github.com/user-attachments/assets/c50f5384-bb34-4058-becf-b8658a858fef)

npm test AddProject.test.jsx
![AddProject test file](https://github.com/user-attachments/assets/2d6e296d-c4a1-446c-a9d1-55edc321c43b)


npm test Radio.test.jsx
![Radio test file](https://github.com/user-attachments/assets/fe8db664-71b7-46b5-9309-b9319d010065)


npm test Selectors.test.js

![Selectors test file](https://github.com/user-attachments/assets/7dce20ff-d50f-4a8c-993c-567ea9dc1306)

npm test TasksTable.test.js

![TaskTables test file](https://github.com/user-attachments/assets/757c379e-5aea-4f03-bf85-78fffbb909fc)
